src/assets/emails/signup-welcome.mjml

```diff
@@ -26,19 +26,19 @@
 
         <mj-button
           mj-class="btn"
-          href="{{apiUrl}}/account/verifyEmail/{{signupToken}}"
+          href="{{mailLinkUrl}}"
```

Comment:
@SobakaSlava let's use a different variable name for link urls, to avoid confusion when there will be more than one link in email

examples:

signup-welcome.mjml
mailLinkUrl => verifyEmailUrl

forgot-password.mjml
mailLinkUrl => resetPasswordUrl

Reply:
@IharKrasnik  Good point, will implement